### DISCUSSION TITLE
back money

### DISCUSSION THREAD

**Discussion by yushuangfei** [community/community#69513] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

After the free one month trial of GitHub Copilot has ended, it will automatically deduct the annual fee for me. Currently, I am not using it. Can you refund the deducted amount to me, and the refund part is also acceptable

**Comment by davecheney** [comment, marked as answer]
@yushuangfei we cannot process refund requests via this forum. Please contact support https://support.github.com/contact.
